content/tutorials/kit-assembly.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Connecting your kit
+---
+
+## The bits you should have
+- Raspberry Pi (the board in a plastic case with many USB ports)
+- Power Board (the board in a plastic case with many green sockets and two buttons)
+- Motor Board (the other board in a plastic case)
+- Arduino (The blue, exposed board with a metal USB-B connector)
+- Arduino Servo Shield (another blue board, hopefully on top of the first)
+- 1 x 3.81mm Camcon (for the Arduino Servo Shield)
+- 2 x 5mm Camcon (for the Motors)
+- 1 x 5mm Camcon with wire loop (attached to the power board)
+- 2 x 7.5mm Camcon (for the Motor Board)
+- USB Hub
+- USB Webcam
+- 2 x Black USB-A to micro-usb cables
+- Beige USB-A to USB-B cable (for the Arduino)
```

Comment:
I'd suggest to removing the colours from this line and the one above - we can (and quite possibly will) give them any old colour of USB cable, and it can be confusing if the cable they're holding doesn't match the description of it this page.